WeakAuras/Prototypes.lua

```diff
@@ -6215,6 +6215,47 @@ WeakAuras.event_prototypes = {
     automaticrequired = true
   },
 
+  ["Current Action"] = {
+    type = "status",
+    events = {
+      ["events"] = {"ACTIONBAR_UPDATE_STATE"}
+    },
+    internal_events = {
+      "ACTIONBAR_SLOT_CHANGED",
+      "ACTIONBAR_PAGE_CHANGED"
+    },
+    name = L["Current Action"],
+    init = function(trigger)
+      local _, _, icon, _, _, _, spellid = GetSpellInfo(trigger.spellName)
+      local button
+      if icon and spellid then
+          icon = icon
+          local slotList = C_ActionBar.FindSpellActionButtons(spellid)
```

Comment:
This is a neat API. Do you happen to know how it handles macros?

Reply:
On my warrior i only have a macro for heroic strike in my action bars and it works

Reply:
Cool. We might be able to actually give people a solution when they want their auras to have keybind text.

Reply:
Also, add C_ActionBar to .luacheckrc